adafruit_funhouse/network.py

```diff
@@ -120,6 +120,14 @@ def mqtt_loop(self):
         except MQTT.MMQTTException as err:
             print("MMQTTException: {0}".format(err))
 
+    def mqtt_publish(self, *args, **kwargs):
```

Comment:
Because this works with both mini mqtt and io mqtt and the publish methods have different arguments. See:
https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/blob/master/adafruit_minimqtt/adafruit_minimqtt.py#L556
and
https://github.com/adafruit/Adafruit_CircuitPython_AdafruitIO/blob/master/adafruit_io/adafruit_io.py#L372

Reply:
Ok, I'll leave it for now...I disagree with using a "global" function for publishing via two separate libraries, users may get confused between the syntax of providing a feed key and an MQTT topic (differently formatted strings).

Reply:
Ideally, we'd have `io_publish` and `mqtt_publish` to really split it out.

Reply:
Fair enough. I was trying to keep the code from growing too big, but may eventually end up going that route.